frontend/src/components/GamePage.jsx

```diff
@@ -17,14 +17,17 @@ const GamePage = ({gameMap}) => {
 
 const TopBar = ({user, logout}) => {
     return (<div style={{display: 'flex', justifyContent: 'space-around'}}>
-        <h1>Project Brooklyn</h1>
-        <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+        <div>
+            <h1>Project Brooklyn</h1>
+            <h2>A tower defense game</h2>
```

Comment:
```suggestion
            <h2>A 3D Tower Defense Game</h2>
```
just a little extra sauce

Reply:
Why do you think we should highlight the 3D?

Reply:
For me, that sets it apart the most. I played a lot of 2D games like this (in Starcraft 1 Use Map Settings scenarios), and it would be all about building a maze to delay the enemies, but there was always a hard rule that you couldn't wall the path off completely (or they would kill all your towers or something). So in my mind, the climbing makes this "not just another tower defense", but that's definitely something that we can discuss, or reach out for feedback on.

Reply:
Makes sense. I don't have a strong opinion one way or the other, I think I was just going for the fewest words possible to make the type of game clear since we don't have instructions yet.

I also got my start in tower defense through Starcraft as well, at least recognizing it as that genre. I just discovered an interesting history of the genre on YouTube, although he didn't mention StarCraft for some reason: https://www.youtube.com/watch?v=7KRszo2vZ5Y

Luckily someone commented about that oversight. There's apparently a Wikipedia page on the genre too. Turns out capitalized "Tower Defense" is [trademarked](https://en.wikipedia.org/wiki/Tower_defense#USPTO_trademark), lol.